includes/models/model.llms.add-on.php

```diff
@@ -331,6 +331,29 @@ public function get_status( $translate = false ) {
 
 	}
 
+	/**
+	 * Get the addon or author image URL for the add-on.
+	 *
+	 * @since [version]
+	 *
+	 * @param string $type Type of image to retrieve. Defaults to null. Accepts "addon" or "author".
+	 * @return string
+	 */
+	public function get_image( $type = null ) {
+
+		$img = esc_url( 'addon' === $type ? $this->get( 'image' ) : $this->get( 'author' )['image'] );
```

Comment:
Let's escape the url in the template, also revert the logic so that if you supply a type differnt from author it always defaults on the addon image.

```suggestion
		$img = 'author' === $type ? $this->get( 'author' )['image'] : $this->get( 'image' );
```

Reply:
got it 👌